packages/client/src/App.vue

```diff
@@ -46,4 +49,17 @@ body {
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
 }
+
+.stream-overlay {
+  display: flex;
+  width: 1920px;
+  height: 1080px;
+
+  &.isDev {
+    background-image: url("./assets/background-dev.jpg");
```

Comment:
do we know if that works? https://vitejs.dev/guide/assets.html

Reply:
now the ".is-dev" class is working in prod :D